### PR TITLE
feat(security): add security as a content type

### PR DESCRIPTION
**Pull request by jakebolam**
Implements https://github.com/kentcdodds/all-contributors/issues/106

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -5,20 +5,17 @@
 "@mrmlnc/readdir-enhanced@^2.2.1":
   version "2.2.1"
   resolved "https://registry.yarnpkg.com/@mrmlnc/readdir-enhanced/-/readdir-enhanced-2.2.1.tgz#524af240d1a360527b730475ecfa1344aa540dde"
-  integrity sha512-bPHp6Ji8b41szTOcaP63VlnbbO5Ny6dwAATtY6JTjh5N2OLrb5Qk/Th5cRkRQhkWCt+EJsYrNB0MiL+Gpn6e3g==
```
Comment (jakebolam) [inline yarn.lock]: Whoops ran with yarn 1.9.2, will revert this

File: README.md [hunk 2]
```diff
@@ -86,6 +86,7 @@ Emoji | Represents | Links to | Comment
 📦 | Packaging/porting to support a new platform
 🔌 | Plugin/utility libraries | the repo home
 👀 | Reviewed Pull Requests
+🛡️ | Security
```
Comment (kentcdodds) [inline README.md]: What type of contribution is security exactly? I mean, what would someone do to make a security contribution?

Comment (jakebolam) [reply README.md]: Great question! I didn't want to put a description to narrow the field (altho maybe we want that?)

Along the lines of, conducts security reviews, helped identify and reduce security threats of the project. Could even tie into GDPR and privacy

Comment (jakebolam) [reply README.md]: Updated readme

**Pull request merged**